### DISCUSSION TITLE
Scale resolution real pixels?

### DISCUSSION THREAD

**Discussion by yingtian991221** [alvr-org/ALVR#1132] [Q&A] [answered]
I'm using Quest 2 and my PC is AMD 5900HX cpu, RTX 3080 laptop gpu, 32GB RAM. I can feel the differences when I change the scales of encoding or rendering resolution from 0.25 to 1.0, but can anyone give me the details of how many pixels are really encoded and rendered in these settings, or how is the scale calculated back into the absolute resolution?

**Comment by zmerp** [comment, marked as answer]
The multiplier/percentage is relative to the default resolution provided by the headset VR runtime, which is arbitrary and usually lower than the screen resolution of the headset. ALVR then multiplies this resolution by the multiplier in the settings, reduces the resolution to a multiple of 32 for the height and a multiple of 64 for the width spanning both eyes (this is done for the encoding alignment requirement on nvidia), the image may be compressed further if using foveated encoding. The client undoes these transformations and alignments before rendering the frame.
